index.html

```diff
@@ -290,13 +321,14 @@ <h2 id="connect">Let's Connect</h2>
           </button>
         </div>
       </li>
+    </ul>
+    <ul class="catagories-section">
```

Comment:
- [ ] [  **OPTIONAL** ] APPLICABLE FOR DESKTOP VERSION: I think that the categories section seems to be left-aligned more than required. It would look great if you could fix the alignment issue by referring to the Figma template 👍🏽  

Your design 👇🏽 
<img width="1440" alt="Screenshot 2022-12-10 at 1 34 18 AM" src="https://user-images.githubusercontent.com/24832846/206790476-c7a97074-e69b-41ed-9eb4-aec681eb1a0f.png">


Figma design 👇🏽 
<img width="1132" alt="Screenshot 2022-12-10 at 1 39 06 AM" src="https://user-images.githubusercontent.com/24832846/206787995-dff3af13-ab26-44bf-8db4-92f718326c64.png">